CHANGELOG.md

```
@@ -9,42 +9,39 @@
 
 ## [0.1.1](https://github.com/VilnaCRM-Org/website/compare/v0.1.0...v0.1.1) (2025-06-07)
 
-
 ### Performance Improvements
 
-* **#35:** improve mobile performance in Lighthouse ([#183](https://github.com/VilnaCRM-Org/website/issues/183)) ([cc714c4](https://github.com/VilnaCRM-Org/website/commit/cc714c4cee6ef1240653266462f452998eb1d27d)), closes [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35)
-
-
+- **#35:** improve mobile performance in Lighthouse ([#183](https://github.com/VilnaCRM-Org/website/issues/183)) ([cc714c4](https://github.com/VilnaCRM-Org/website/commit/cc714c4cee6ef1240653266462f452998eb1d27d)), closes [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35) [#35](https://github.com/VilnaCRM-Org/website/issues/35)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)